src/parquet/arrow/arrow-reader-writer-test.cc

```diff
@@ -325,6 +330,73 @@ void WriteTableToBuffer(const std::shared_ptr<Table>& table, int num_threads,
   *out = sink->GetBuffer();
 }
 
+namespace internal {
```

Comment:
Can we move this code into Arrow after we have done the releases?

Reply:
Yes, opened https://issues.apache.org/jira/browse/ARROW-1929 so as to not forget